superset-frontend/src/views/CRUD/data/database/types.ts

```diff
@@ -25,6 +25,7 @@ export type DatabaseObject = {
   // Connection + general
   id?: number;
   database_name: string;
+  name: string; // synonym to database_name
```

Comment:
We're setting `name` instead of `database_name` somewhere, this is just a quick fix.